### PR TITLE
Build/split

### PR DESCRIPTION
**Pull request by SimonSchick**
- Removed webpack build process
- Build commonjs AND modules
- Cleaned up some stuff
- No more including `package.json`, less bytes!

### CODE REVIEW

**Comment by connor4312** [comment]
General: might be nice to add browser tests in here too

**Comment by SimonSchick** [comment]
Out of scope, just trying to get borm building again with module packaging.

@Antariano worked on a test stack and would love to re-use them in time.

File: test/socket.test.js [hunk 1]
```diff
@@ -18,7 +18,10 @@ describe('socket', () => {
             socket = null;
         }
 
-        server.close(done);
+        server.close(err => {
+            // delay for freeing up port
```
Comment (connor4312) [inline test/socket.test.js:22]: How about making the socket give a callback or event when it closes?

**Review by connor4312 — APPROVED** [review]
Seems aright overall

File: test/socket.test.js (hunk 1 above)
Comment (SimonSchick) [reply test/socket.test.js:22]: Isn't that what the callback is for?

I just delayed the callback a little more.

Comment (connor4312) [reply test/socket.test.js:22]: Erp, yea, sorry, reading during meetings. Kinda unfortunate that (presumably?) the callback fires before the listener is actually closed.

Comment (SimonSchick) [reply test/socket.test.js:22]: We should look into update to ws@2.

**Pull request merged**